src/ophyd_async/epics/_backend/_aioca.py

```diff
@@ -126,7 +187,7 @@ def make_converter(
             datatype_dtype = get_dtype(datatype)
             if not datatype_dtype or not np.can_cast(datatype_dtype, np.str_):
                 raise TypeError(f"{pv} has type [str] not {datatype.__name__}")
-        return CaArrayConverter(pv_dbr, None)
+        return CaConverter(pv_dbr, None)
```

Comment:
So these have to go back too

Reply:
The last test case for `test_backend_get_put_monitor()` in `test_signals.py` fails when using the array converter because a value isn't passed through, and I wasn't sure how that was meant to occur.
This case that was added with the last commit to main.
```
(
            npt.NDArray[np.str_],
            "stra",
            ["five", "six", "seven"],
            ["nine", "ten"],
            {"ca"},
        ),
```
I was advised this is the way to go now, i'm not sure which way to solve this.

Reply:
I have put it back.